src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -1436,6 +1445,18 @@ public long GetEnvironmentsHash()
 
             return hash;
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public void AddToCurrentlyRunningGraphQueries(ExecutingQueryInfo executingQueryInfo)
+        {
+            CurrentlyRunningGraphQueries.Add(executingQueryInfo);
```

Comment:
Can we move it to GraphQueryRunner and acces it via Database.QueryRunner.Graph.CurrentlyRunningGraphQueries ?